### PR TITLE
Add documentation for new Investment Lots report

### PR DESCRIPTION
**Pull request by mcbridebt**
Report was added to master here: https://github.com/Gnucash/gnucash/pull/1464

### CODE REVIEW

File: C/manual/ch_Reports.xml [hunk 1]
```diff
@@ -2086,6 +2086,722 @@ Income (type INCOME)
...
+
+                <imageobject role="fo">
+                  <imagedata fileref="figures/Report_Investment_Lots.png"
+                             srccredit="Brent McBride"/>
+                </imageobject>
+              </mediaobject>
+            </screenshot>
+          </figure>
+
+          <para>The Investment Lots report shows details about lots in a selected set of accounts of type
```
[the remark this answers is not in the document]
Comment (mcbridebt) [reply C/manual/ch_Reports.xml:2111]: You mean like this?
```
<abstract>
  <para>The Investment Lots report ...
  </para>
</abstract>
```
Wouldn't that make this section inconsistent with the rest of the doc? I don't see another usage of <abstract> in the file.

[the remark this answers is not in the document]
Comment (fellen) [reply C/manual/ch_Reports.xml:2111]: It is one of the targets to improve the docs, see [Documentation_Improvement#Maintenance](https://wiki.gnucash.org/wiki/Documentation_Improvement#Maintenance).

File: C/guide/ch_reports.xml [hunk 2]
```diff
@@ -304,6 +304,67 @@
...
+          </listitem>
+
+          <listitem>
+            <para>Are the lots (and related accounts and splits) well-formed?
+            </para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For more details, see the
+          <ulink url="&url-docs;/C/gnucash-help/report-classes.html#investment-lots-report">Investment Lots</ulink>
```
Comment (fellen) [inline C/guide/ch_reports.xml:363]: 1. Please use here and later the entities from [docbook/gnc-docbookx.dtd#L119-L124](https://github.com/Gnucash/gnucash-docs/blob/master/docbook/gnc-docbookx.dtd#L119-L124)
2. `help` should be called `manual` now.

**Comment by mcbridebt** [comment]
Thanks for the feedback! I resolved the issues. Anything else I should change, or can this PR be merged?

**Comment by mcbridebt** [comment]
Can this PR be merged now? The new report is already out in the latest pre-release build, so it would be good to have its documentation published, too.

**Review by fellen — APPROVED** [review]
Best existing report description!

I will merge it now. Please let me know your opinion on the itemizedlist -> glosslist convrsion.

File: C/manual/ch_Reports.xml [hunk 3]
```diff
@@ -2086,6 +2086,724 @@ Income (type INCOME)
...
+            <para>The Amount columns are denominated in each account's commodity. But the Value, Basis, Price, and
+            Gain columns are always denominated in the <guilabel>Report's currency</guilabel>, converted if necessary.
+            </para>
+          </note>
+          <itemizedlist>
+            <listitem  id="investment-lots-report-column-title">
+              <para><guilabel>Lot Title</guilabel>
+              </para>
+            </listitem>
+            <listitem id="investment-lots-report-column-guid">
```
[the remark this answers is not in the document]
Comment (mcbridebt) [reply C/manual/ch_Reports.xml:2182]: I'll trust your judgement on that. This was my first attempt writing documentation in this XML format, so I don't know what is the difference between <itemizedlist> and <glosslist>

[the remark this answers is not in the document]
Comment (fellen) [reply C/manual/ch_Reports.xml:2182]: > I don't know what is the difference between itemizedlist and glosslist

Compare the appearance of the lists e.g. between
https://code.gnucash.org/docs/C/gnucash-help/tool-price.html with
https://code.gnucash.org/docs/de/gnucash-help/tool-price.html
and  its readability in the sources section `<sect1 id="tool-price">` in each ch_Tools_Assistants.xml.

**Pull request merged**